UnityProject/Assets/Scripts/UI/Core/ControlTabs.cs

```diff
@@ -554,12 +561,7 @@ public static void CheckTabClose()
 
 			foreach (NetTab tab in toClose)
 			{
-				Instance.HideTab(tab);
-			}
-
-			foreach (NetTab tab in toDestroy)
-			{
-				Instance.DestroyTab(tab);
+				Instance.DestroyTab(tab);;
 			}
```

Comment:
What happens to the tabs in `toDestroy`?

Reply:
yeah, got lost in conflicts